routes/modelExport.js

```diff
@@ -1,9 +1,33 @@
-var linz = require('../');
+'use strict';
+
+const linz = require('../');
+const setTemplateScripts = require('../lib/scripts');
```

Comment:
@smebberson How would a programmer use it though?

I actually though about something like this to set `res.locals` and expose that through the api. That is probably best done in another PR though since it would require refactoring of a lot of code that isn't related to `scripts` and `styles`?